src/hotspot/cpu/aarch64/continuationHelper_aarch64.inline.hpp

```diff
@@ -68,6 +68,14 @@ inline void ContinuationHelper::push_pd(const frame& f) {
   *(intptr_t**)(f.sp() - frame::sender_sp_offset) = f.fp();
 }
 
+inline address ContinuationHelper::return_pc_at(intptr_t* sp) {
+  return pauth_strip_pointer(*(address*)sp);
```

Comment:
This is the return address. it's called `return_address` elsewhere.

Reply:
I used `return_pc_at()` following the usage of `ContinuationHelper::Frame::return_pc()` and `ContinuationHelper::InterpretedFrame::return_pc()`.

How about `get_pc_at()` or `return_address_at()`? 
Thanks.

Reply:
Rename to `return_address_at()` in the latest revision. Thanks.